cmd/flow-aggregator/config.go

```diff
@@ -48,4 +48,7 @@ type FlowAggregatorConfig struct {
 	// is not available), a value will be randomly generated, which may vary across restarts of the flow
 	// aggregator.
 	ObservationDomainID *uint32 `yaml:"observationDomainID,omitempty"`
+	// Provide format for record sent to the configured flow collector. Supported formats are IPFIX and JSON.
```

Comment:
s/Provide format for record/Provide format for records

same for other occurrences